File: PYTHON/Program/Human.py
class Human:
    def __init__(self, name="", id="", partai="", bidang=""):
        self.name = name
        self.id = id
        self.partai = partai
        self.bidang = bidang

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

    def get_partai(self):
        return self.partai

    def set_partai(self, partai):
        self.partai = partai

    def get_bidang(self):
        return self.bidang

    def set_bidang(self, bidang):
        self.bidang = bidang

File: JAVA/Program/Human.java
public class Human {
    private String name;
    private String id;
    private String partai;
    private String bidang;

    public Human() {
        this.name = "";
        this.id = "";
        this.partai = "";
        this.bidang = "";
    }

    public Human(String name, String id, String partai, String bidang) {
        this.name = name;
        this.id = id;
        this.partai = partai;
        this.bidang = bidang;
    }

    public String get_name() {
        return this.name;
    }

    public void set_name(String name) {
        this.name = name;
    }

    public String get_id() {
        return this.id;
    }

    public void set_id(String id) {
        this.id = id;
    }

    public String get_partai() {
        return this.partai;
    }

    public void set_partai(String partai) {
        this.partai = partai;
    }

    public String get_bidang() {
        return this.bidang;
    }

    public void set_bidang(String bidang) {
        this.bidang = bidang;
    }
}